src/Sarif.Viewer.VisualStudio.Core/Converters/AnalysisStepNodeToTextConverter.cs

```diff
@@ -11,6 +11,8 @@ namespace Microsoft.Sarif.Viewer.Converters
 {
     internal class AnalysisStepNodeToTextConverter : IValueConverter
     {
+        internal static readonly char IndentChar = ' ';
```

Comment:
>internal static readonly char IndentChar = ' '; [](http://example.com/codeflow?start=8&length=47)

Have there been any accessibility considerations for the single space indent? Looking at the screenshot, I only registered the indents because I read the description and code.

Reply:
updated the solution to set indent by setting left margin of the text. It can be more accurate to pixel level and doesn't affect text content itself. 

Updated the screenshot in description. Please take a look and let me know what do you think

Reply:
Awesome. So much easier for me to read. @EasyRhinoMSFT, can you take a look at this?

Reply:
Looks good!

Reply:
Looks good! Using a margin is the right way to go.